kempner_hpc_handbook/development_and_runtime_envs/using_conda_env.md

```diff
@@ -24,10 +24,10 @@ Overall, conda environments allow you to isolate package versions for different
 
 - Step 2: Purge the loaded modules using the `module purge` command if you have any loaded modules and you want to start fresh.
 - Step 3: Load `python` module using the `module load python` command. This will load the default version of Python.
-- Step 5: Create a conda environment using the `mamba create` command. For example, to create a conda environment named `myenv` with Python 3.12 and installing pip and numpy, you can use the following command:
+- Step 5: Create a conda environment using the `mamba create` command. For example, to create a conda environment named `myenv` with Python 3.12 and installing pip and numpy from the channel conda-forge, you can use the following command:
```

Comment:
Thank you, @dmbala. 
The default channel will be sufficient for the current packages. How about adding a new tip box mentioning that you can add a channel to the command if needed, or give the highest priority to the conda-forge channel using `conda config --add channels conda-forge`?
Here is an example of tip: https://github.com/KempnerInstitute/kempner-hpc-handbook/blob/88471a51737b441a5111bfab11583ae369841ac0/kempner_hpc_handbook/resource_management/array_jobs.md?plain=1#L17C1-L19C4